SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestAttachedComponentReplication/SpatialTestAttachedComponentReplication.cpp

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright 1998-2019 Epic Games, Inc. All Rights Reserved.
+// Copyright 1998-2019 Epic Games, Inc. All Rights Reserved.
```

Comment:
Huh, why does this have Epic's copyright header? Isn't this our code?

Reply:
huh, good point